Assets/Scripts/Behaviours/SteeredCohesionBehaviour.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(menuName = "Flock/Behaviour/Steered Cohesion")]
+public class SteeredCohesionBehaviour : FilteredFlockBehaviour
+{
+    private Vector3 currentVelocity;
```

Comment:
It would be better if that velocity was fetched from the flock object, non?

Reply:
Yeah this is something that will be done with the `Kinematic` class, currently not in my scope

Reply:
alright @Taqi take note of that 😛